templates/multipass/install.html

```diff
@@ -0,0 +1,329 @@
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <p class="p-heading--5">Clean up what you don't need:</p>
+              </div>
+              <div class="col-6 col-medium-3">
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass delete bar</code></pre>
+                </div>
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass purge</code></pre>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <p class="p-heading--5">Find alternate images to launch:</p>
+              </div>
+              <div class="col-6 col-medium-3">
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass find</code></pre>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <p class="p-heading--5">
+                  Pass a cloud-init metadata file to an instance on launch. (See <a href="https://blog.ubuntu.com/2018/04/02/using-cloud-init-with-multipass">using cloud-init with multipass</a> for more details):
+                </p>
+              </div>
+              <div class="col-6 col-medium-3">
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass launch -n bar --cloud-init cloud-config.yaml</code></pre>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-3 col-medium-3">
+                <p class="p-heading--5">Get help:</p>
+              </div>
+              <div class="col-6 col-medium-3">
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass help</code></pre>
+                </div>
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block--icon"><code>multipass help &lt;command&gt;</code></pre>
+                </div>
+              </div>
+            </div>
+          </li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-start-large-4 col-6 col-start-medium-4 col-medium-3">
+            <p>
+              Don't forget you still have the "foo" instance running! To learn more about Multipass keep reading, <a href="/docs">go to the docs</a>, or <a href="https://discourse.ubuntu.com/c/multipass/21">join the discussion and get involved.</a>
```

Comment:
```suggestion
              Don't forget you still have the "foo" instance running. To learn more about Multipass, <a href="multipass/docs">go to the docs</a> or <a href="https://discourse.ubuntu.com/c/multipass/21">join the discussion and get involved.</a>
```

Reply:
Is /multipass/docs being added to the feature branch?

Reply:
It's already there, I just forgot to update the link here after rebasing 😅